Serpent/PythonBuilder.cs

```diff
@@ -145,10 +166,13 @@ private Python Build(ReadOnlyMemory<byte>? python)
             env.TryAdd("PYTHONDONTWRITEBYTECODE", "1");
             env.TryAdd("PYTHONUNBUFFERED", "1");
 
-            // Specify main.py if some code was supplied, otherwise start in interactive mode
-            var environment = python.HasValue
-                            ? new BasicEnvironment(env, ["python", "main.py"])
-                            : new BasicEnvironment(env, ["python"]);
+            // Ensure the main file path is set.
+            _mainFilePath ??= "main.py";
```

Comment:
You can set this in the field initializer, that way `_mainFilePath` can be `string` instead of `string?`